infrasim/model.py

```diff
@@ -718,7 +718,7 @@ def handle_parms(self):
         if self.__product:
             self._dev_attrs["product"] = self.__product
 
-        if self.__rotation:
```

Comment:
@XiaowenJiang I see in SCSIDrive.init, the rotation can be an empty string. In this condition, this will assign self._dev_attrs["rotation"] to empty string, which makes add_option eventually leave an option in qemu like "...,rotation=,...", this will crash qemu.

Original statement can handle empty string condition.